### PR TITLE
Add custom font example

### PR DESCRIPTION
**Pull request by piepie62**
Depends on mkbcfnt and recent unreleased libctru and citro2d functions

### CODE REVIEW

File: graphics/printing/custom-font/source/main.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+static void sceneExit(void)
+{
+	// Delete the text buffers
+	C2D_TextBufDelete(g_staticBuf);
+}
+
+int main()
+{
+	romfsInit();
+	sdmcInit();
```
Comment (fincs) [inline graphics/printing/custom-font/source/main.c]: This is unnecessary

**Comment by oreo639** [comment]
https://github.com/smealum/ctrulib/blob/d1ca5c4ea6f08a4a2b06659a4633c9acaeaaccdb/libctru/source/system/appInit.c
sdmcInit() is automatically done when you start a regular application. romfsInit() is not.
I think you still need to add back romfsInit() as I believe fincs was just talking about sdmcInit().
Correct me if I am wrong.

**Comment by fincs** [comment]
Correct. I only wanted the sdmcInit call to be removed.

File: graphics/printing/custom-font/source/main.c [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	C2D_DrawText(&g_staticText[0], 0, 8.0f, 8.0f, 0.5f, size, size);
+	C2D_DrawText(&g_staticText[1], C2D_AtBaseline, 108.0f, 36.0f, 0.5f, size, size);
+	C2D_DrawText(&g_staticText[2], C2D_AtBaseline, text2PosX, 210.0f, 0.5f, size, size);
+}
+
+static void sceneExit(void)
+{
+	// Delete the text buffers
+	C2D_TextBufDelete(g_staticBuf);
+}
```
Comment (fincs) [inline graphics/printing/custom-font/source/main.c:48]: Missing font cleanup here.

File: graphics/printing/custom-font/source/main.c [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	}
+
+	// Deinitialize the scene
+	sceneExit();
+
+	// Deinitialize the libs
+	C2D_Fini();
+	C3D_Fini();
+	gfxExit();
+	return 0;
```
Comment (fincs) [inline graphics/printing/custom-font/source/main.c:95]: cfguExit
romfsExit

**Pull request merged**